packages/twenty-front/src/modules/settings/components/SettingsOptionCardContent.tsx

```diff
@@ -1,34 +1,76 @@
+import { SettingsCounter } from '@/settings/components/SettingsCounter';
+import { Select } from '@/ui/input/components/Select';
 import { useTheme } from '@emotion/react';
 import styled from '@emotion/styled';
-
 import { useId } from 'react';
 import { CardContent, IconComponent, Toggle } from 'twenty-ui';
 
-type SettingsOptionCardContentProps = {
+type BaseSettingsOptionCardContentProps = {
   Icon?: IconComponent;
   title: React.ReactNode;
-  description: string;
+  description?: string;
   divider?: boolean;
+  disabled?: boolean;
+  advancedMode?: boolean;
+};
+
+type ToggleProps = {
+  variant: 'toggle';
   checked: boolean;
   onChange: (checked: boolean) => void;
 };
 
-const StyledCardContent = styled(CardContent)`
+type CounterProps = {
```

Comment:
yes let's split this into 3 components. We can introduce a subfolder: option-card in settings